css-shared-element-transitions/index.bs

```diff
@@ -298,241 +299,302 @@ Single-page API:
+
+    1. For each [=fully active=] {{Document}} in <var ignore>docs</var>,
+        [=perform pending transition operations=] for that {{Document}}.
+
+    Note: These steps will be added to the [=update the rendering=] in the HTML spec.
+        As such, the prose style is written to match other steps in that algorithm.
+</div>
+
+<div algorithm="perform pending transition operations">
+    To <dfn>perform pending transition operations</dfn> given a {{Document}} |document|,
+        perform the following steps:
+
+    1. If |document|'s [=document/pending same document transition outgoing capture=] is not null, then:
+
+        1. [=Perform an outgoing capture=] with |document|'s [=document/pending same document transition outgoing capture=].
+
+        1. Set |document|'s [=document/pending same document transition outgoing capture=] to null.
+
+    1. If |document|'s [=document/running same document transition=] is not null,
+        then [=update transition DOM=] for |document|'s [=document/running same document transition=].
+</div>
+
+<div algorithm="perform outgoing capture">
+    To <dfn>perform an outgoing capture</dfn> given a {{SameDocumentTransition}} |transition|,
+        perform the following steps:
+
+    1. Let |taggedElements| be |transition|'s [=SameDocumentTransition/tagged elements=].
+
+    1. Let |usedTransitionTags| be a new [=/set=] of strings.
+
+    1. Let |preparationFailed| be false.
+
+    1. Let |document| be |transition|'s [=relevant global object's=] [=associated document=].
+
+    1. [=list/For each=] |element| of every {{Element}} and [=pseudo-element=] connected to |document|,
+        in [paint order](https://drafts.csswg.org/css2/#painting-order):
+
+        Issue: The link for "paint order" doesn't seem right.
+            Is there a more canonical definition?
+
+        1. Let |transitionTag| be the [=computed value=] of 'page-transition-tag' for |element|.
+
+        1. If |transitionTag| is ''page-transition-tag/none'',
+            or |element| is [=element-not-rendered|not rendered=],
+            then [=continue=].
+
+        1. If any of the following is true:
+
+            * |usedTransitionTags| [=list/contains=] |transitionTag|.
+
+            * |element| is not |element|'s [=tree/root=] and |element| does not have [=layout containment=].
+
+            * |element| is not |element|'s [=tree/root=] and |element| does prevent [=fragmentation=].
+
+            Then set |preparationFailed| to true and [=break=].
+
+        1. [=set/Append=] |transitionTag| to |usedTransitionTags|.
+
+        1. Let |capture| be a new [=captured element=] struct.
+
+        1. Set |capture|'s [=outgoing image=] to the result of [=capturing the image=] of |element|.
+
+        1. Set |capture|'s [=outgoing styles=] to the following:
+
+            : 'transform'
+            :: A CSS transform that would place |element|
+                from the document origin to its current quad.
```

Comment:
```suggestion
                from the layout viewport origin to its current quad.
```

Since we're positioning all shared elements wrt to the layout viewport.